app/src/main/AndroidManifest.xml

```diff
@@ -224,6 +224,18 @@
             android:enabled="true"
             android:exported="false"
             android:label="SyncReceiver" />
+
```

Comment:
could you explain more context on this FCM usage?
if we need to display notification, we need to add notification permission since API 33 and add notification channel since API 26